pkg/vm/slot.go

```diff
@@ -23,13 +23,8 @@ func (s slot) Set(i int, item stackitem.Item, refs *refCounter) {
 	if s[i] == item {
 		return
 	}
-	old := s[i]
+	refs.Remove(s[i])
 	s[i] = item
-	if old != nil {
-		refs.Remove(old)
-	} else {
-		*refs-- // Not really existing, but counted Null element.
```

Comment:
But how about this Null element?

Reply:
`refs.Remove(nil)` which works just fine, decrementing the counter.

Reply:
Are you sure that `refs.Remove(nil)` decrements the counter?

Reply:
Yeah, why not?

Reply:
Ugh, I’ve misread the code, everything is OK here, but tests need to be fixed.

Reply:
Fixed.